.gemini/config.yaml

```diff
@@ -0,0 +1,20 @@
+# Minimize verbosity.
+have_fun: false
+code_review:
+  # For now, use the default of MEDIUM for testing. Based on desired verbosity,
+  # we can change this to LOW or HIGH in the future.
+  comment_severity_threshold: MEDIUM
+  pull_request_opened:
+    # Explicitly set help to false in case the default changes in the future, as
+    # having a help message on every PR would be spammy.
+    help: false
+    # These tend to be verbose, and since we expect PR authors to clearly
+    # describe their PRs this would be at best duplicative.
+    summary: false
+ignore_patterns:
+  # Avoid code reviews on rolls.
+  - DEPS
+  - bin/internal/**
```

Comment:
is `bin/internal` a roll?  that's our shell scripts (bash/powershell).

Reply:
It includes .version files that are rolled. For example https://github.com/flutter/flutter/pull/172696.

Reply:
I will update the pattern to include `.version`